libevmasm/Instruction.h

```diff
@@ -89,6 +89,7 @@ enum class Instruction: uint8_t
 	CHAINID,			///< get the config's chainid param
 	SELFBALANCE,		///< get balance of the current account
 	BASEFEE,            ///< get the block's basefee
+	BLOBHASH = 0x49,    ///< get a versioned hash of one of the blobs associated with the transaction
```

Comment:
You don't need an explicit value for this - `BASEFEE` is `0x48`. Important to note - if `blobbasefee` ends up getting merged first, it would be nice if you could get rid of the `0x4a` assignment when rebasing.

Reply:
We don't, but I think it's better to have the explicit value. Less error prone than counting the values and easier to see the value when you need it (e.g. in review).

Reply:
I agree, but then we should do so for all of them; obviously not in this PR. I suggested he remove it for the sake of consistency.

Reply:
I don't think we'll ever do this properly so I'd rather leave existing entries as is and just add values to new or modified ones as we change them over time.